src/commands/synthetics/compatibility.ts

```diff
@@ -24,3 +24,18 @@ export const replaceGlobalWithDefaultTestOverrides = (
 
   return config
 }
+
+export const replaceConfigWithTestOverrides = (
+  config?: UserConfigOverride,
+  testOverrides?: UserConfigOverride
+): UserConfigOverride => {
+  const isConfigUsed = Object.keys(config ?? {}).length !== 0
+  const isTestOverridesUsed = Object.keys(testOverrides ?? {}).length !== 0
+
+  // If both config and testOverrides exist use testOverrides
```

Comment:
**nit** add a comma to make it breathe 😁 (same above)
```suggestion
  // If both config and testOverrides exist, use testOverrides
```

Reply:
> What's the benefit of this parent describe block?

Being able to run all of the test in the file all at once